experiments/azure/cyclecloud/cpu/experiment/data/laghos/logs-08-21-2024/laghos-2n-err.log

```diff
@@ -0,0 +1,3 @@
+real 169.52
```

Comment:
Ditto these scripts - are they from parallel cluster? The 21st was last Wednesday and I don't think we had CycleCloud working yet.

Reply:
They should be removed, and are now.